modules/keycloak/tests/test_keycloak.py

```diff
@@ -3,7 +3,7 @@
 from testcontainers.keycloak import KeycloakContainer
 
 
-@pytest.mark.parametrize("version", ["16.1.1"])
+@pytest.mark.parametrize("version", ["24.0.1"])
```

Comment:
Since we only test for a single parameter, you can probably just inline this version

Reply:
@santi  Thanks for your feedback. I thought about this already myself. Now we already two. 😃 I'll scratch that parametrization.